weave_demo.ipynb

```diff
@@ -0,0 +1,1199 @@
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pantry_name = \"weave-demo-pantry\"\n",
+    "local_fs.mkdir(pantry_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0fc0a17a-1046-4357-8c3d-34ca0a36e7bb",
+   "metadata": {},
+   "source": [
+    "Next let's create a pantry where we will store our basekts. For this pantry we will using Pandas for the index backend. An Index is an object or file that tracks the baskets in a pantry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7d2f65ea-f994-4130-9d1f-9739e988d2d9",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pantry1 = Pantry(IndexPandas, pantry_path=pantry_name, file_system=local_fs)\n",
+    "index_df = pantry1.index.to_pandas_df()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40f50c35-4ea5-4440-897f-1ef41ca7a785",
+   "metadata": {},
+   "source": [
+    "Below is an index dataframe which holds all of the index values for our pantry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "b90fdc4f-a3a2-4b07-bcfa-15b8bc065587",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>uuid</th>\n",
+       "      <th>upload_time</th>\n",
+       "      <th>parent_uuids</th>\n",
+       "      <th>basket_type</th>\n",
+       "      <th>label</th>\n",
+       "      <th>weave_version</th>\n",
+       "      <th>address</th>\n",
+       "      <th>storage_type</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "Empty DataFrame\n",
+       "Columns: [uuid, upload_time, parent_uuids, basket_type, label, weave_version, address, storage_type]\n",
+       "Index: []"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "index_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f3d4f92-ae46-457d-8abf-0023e2015954",
+   "metadata": {},
+   "source": [
+    "## Creating and Uploading Baskets\n",
+    "A Basket is a representation of an atomic data product within a pantry. Below we will demonstrate how to upload a basket containing our dummy data, called WeaveDemoText, using a pantry object. "
```

Comment:
You may want to use this cell to explain your upload in the next file. Why are metadata and basket-type being used? That's a strange upload format (a list of dictionaries of filenames and stubs), what does that mean?